link-checker.js

```diff
@@ -53,12 +53,17 @@ module.exports = function(directory, options = {}, callback) {
 
 	let fileCounter = 0
     debug('scanning directory', directory)
-    
+
     function getOverrideFor(target) {
-        const url = target instanceof Error && target.response ? target.response.request.url // superagent error
-            : typeof target === "object" ? target.request.url // superagent response
-            : typeof target === "string" ? target // plain URL
-            : null
+		const url = target instanceof Error && target.response && target.response.request? target.response.request.url // superagent error
+                  : typeof target === "object" && target.request ? target.request.url // superagent response
+                  : typeof target === "string" ? target // plain URL
+				  : undefined
+
+		if (url === undefined) {
+			console.log(target);
```

Comment:
Feel free to turn this into whatever makes sense, but this is rare and when it happens, it should absolutely break things in my project. That's probably not true for everyone, though.